docs/src/main/tut/symbols.md

```diff
@@ -0,0 +1,19 @@
+| Symbol | Name                   | Typeclass      | Import                       | Source                                              |
+| ------ | ---------------------- | ------------   | --------------------------   | -------------------------------------------------   |
+| `<*>`  | apply                  | `Apply`        | TBA                          | core/src/main/scala/cats/Apply.scala                |
+| `|@|`  | Cartesian builder      | `Cartesian`    | `cats.syntax.cartesian._`    | core/src/main/scala/cats/Cartesian.scala            |
```

Comment:
I also think we could leave out the source location and do pretty well. I'd rather have a column that relates the syntax to the type class, if possible. E.g.

```
| Symbol | Name | | Typeclass | Definition | Import |
|------|------|----------|---------|--------|
| x =!= y | not equals | Eq | Eq[A].eqv(x, y) | cats.syntax.eq |
```